WISE-Community/WISE-Client
# Convert Draw student component to Angular

Make sure student Draw works.

Things to test in particular
- Specific tools can be disabled for the student
- The authoring to allow student to upload background works (functionality existed before but authoring for this is newly added)
- The student can set the background
- The student can add the drawing snapshot as a note (enable the Student Notebook and Enable Clipping)
- The student can add a public note as a background. you can add this to the project JSON to enable public spaces.
```
    "spaces": [
        {
            "id": "public",
            "name": "Public",
            "isPublic": true,
            "isShowInNotebook": true
        }
    ]
```
- The Draw can show work from another Draw
- The Draw can import work from another Draw on a different step
- The Draw can import work from another Draw on the same step on save (functionality existed before but authoring for this is newly added)
- The Draw can import work from another Draw on the same step on submit (functionality existed before but authoring for this is newly added)
- The Draw can import work from Concept Map as a background image
- The Draw can import work from Embedded as a background image
- The Draw can import work from Graph as a background image
- The Draw can import work from Label as a background image
- The Draw can import work from Table as a background image
- Set an authored background on a Draw and have that Draw import work from another Draw that also has a background. the authored background should be the one that is shown to the student.
- When the student clicks reset, the Draw should revert back to the starter state. If that Draw component imports work, it should re-import the work.
- Make sure the Draw grading loads properly
- Make sure the Draw grading only shows the tools the student has access to

Closes #127
